tutorial/2 - Binning of example time-resolved ARPES data.ipynb

```diff
@@ -0,0 +1,192 @@
+    "\n",
+    "sys.path.append(\"../\")\n",
+    "from sed.binning import bin_dataframe"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "42a6afaa-17dd-4637-ba75-a28c4ead1adf",
+   "metadata": {},
+   "source": [
+    "# Load Data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "34f46d54",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataPath = '/path/to/folder' # Put in Path to a storage of at least 20 Gbyte free space.\n",
+    "! curl --output {dataPath}/WSe2.zip https://zenodo.org/record/6369728/files/WSe2.zip\n",
+    "shutil.unpack_archive(dataPath + '/WSe2.zip', extract_dir=dataPath)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0c3be560",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The Scan directory\n",
+    "fdir = dataPath + '/Scan049_1'\n",
+    "dfp = fp.dataframeProcessor(datafolder=fdir)\n",
+    "dfp.read(source='folder', ftype='h5')\n",
+    "ddf=dfp.edf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6902fd56-1456-4da6-83a4-0f3f6b831eb6",
+   "metadata": {},
+   "source": [
+    "# Define the binning range"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a7601cd7-cd51-40a9-8fc7-8b7d32ff15d0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "binAxes = [\"X\", \"Y\", \"t\"]\n",
+    "nBins = [120, 120, 120]\n",
+    "binRanges = [(0, 1500), (0, 1500), (65000, 67000)]\n",
+    "coords = {ax: np.linspace(r[0], r[1], n) for ax, r, n in zip(binAxes, binRanges, nBins)}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ba0416b3-b4b6-4b18-8ed3-a76ab4889892",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ddf.head(10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "01066d40-010a-490b-9033-7339e5a21b26",
+   "metadata": {},
+   "source": [
+    "# compute distributed binning on the partitioned dask dataframe\n",
+    "We generated 100 dataframe partiions from the 100 files in the dataset, which we will bin parallelly with the dataframe binning function into a 3D grid"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cbed3261-187c-498d-8ee0-0c3a3c8a8c1e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "res = bin_dataframe(\n",
+    "    ddf,\n",
+    "    binAxes=binAxes,\n",
+    "    binRanges=binRanges,\n",
+    "    nBins=nBins,\n",
+    "    hist_mode=\"numba\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "99d7d136-b677-4c16-bc8f-31ba8216579c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, axs = plt.subplots(1, 3, figsize=(8, 2.5), constrained_layout=True)\n",
+    "for dim, ax in zip(binAxes, axs):\n",
+    "    res.sum(dim).plot(ax=ax)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4a3eaf0e",
+   "metadata": {},
+   "source": [
+    "# Compare to MPES binning"
```

Comment:
I think this section is not necessary as it again adds confusion on why MPES is doing something SED is supposed to be doing. It might be good for showcasing why we are designing this package, but perhaps it can be in documentation and not the tutorial. What do you think?

Reply:
It's not necessary, but I thought it to be a good crosscheck. We can move it to the tests, maybe, if we incorporate the mpes package into the sed requirements.

Reply:
It definitely is useful to know. I guess it can be left here for now and we can discuss sometime where it can fit better.